.Documentation/articles/innoactive-creator/snapzones.md

```diff
@@ -0,0 +1,96 @@
+- [**Preview**](#preview): The object that will be shown as preview object on the snap zone (Optional).
+
+> By default, a snap zone can interact with any interactable object, to change this behavior (for example, making a snap zone to only accept one type of object), the `Interaction Layer Mask` must be changed in both, the snap zone and interactable object. 
+
+### Snap zone
+
+1. We need to create an empty game object. To do this, right click into the *Unity Scene Hierarchy* and select `Create Empty`.
+2. This empty game object requires a collider component the size that the snap zone should ultimately have. The `Is Trigger` property must be **enabled**. 
+3. It needs the `Snap Zone Property` component. To add it, click on the `Add Component` button in the bottom of the *Unity Inspector*. Then type in `Snap Zone Property` and select the appearing entry.
+
+We can now change the volume color of our snap zone by changing the `Shown Highlight Object Color`. 
+If we want to show a validation preview of the object hovering the snap zone we can make sure the `Show Interactable Hover Meshes` option is enabled. Additionally, we can also customize the appearance of the validation preview by adding a material asset to the `Interactable Hover Mesh Material`.
+
+[![Create Snap Zone Manually](../images/snap-zones/manually-create-snapzone.gif "Create Snap Zone Manually")](../images/snapzones/manually-create-snapzone.gif)
+
+
+### Snappable Object
+
+1. We need a game object that we want to snap into the snap zone. In this example, we create a cube by right clicking into the *Unity Scene Hierarchy* and select `3D Object > Cube`.
+2. Make sure that the game object has a collider component.
+3. To make the game object snappable, we need to add the `Snappable Property` component.
+
+[![Create Snappable Cube](../images/snap-zones/create-snappable-cube.gif "Create Snappable Cube")](../images/snapzones/create-snappable-cube.gif)
+
+
+### Preview
+
+For our preview, we probably want to use a model of our snappable object. If we don't already have a prefab, we can create one by following these steps:
+
+1. To create a preview model of our snappable game object, we can duplicate it by right-clicking on it and selecting `Duplicate`. Afterward, we delete every component but the `Transform`, `Mesh Filter`, `Mesh Renderer`, or `Skinned Mesh Renderer` available in the clone. We also rename the newly created game object to be easily distinguished.
+2. To create the actual prefab, we drag & drop the game object from the *Unity Scene Hierarchy* tab into the desired opened folder of the *Unity Project* tab.
+    > Learn more about [Prefabs](https://docs.unity3d.com/Manual/Prefabs.html)
+3. Now we can add this prefab to the `Show Highlight Object` property in the `Snap Zone` component of our snap zone game object. To do this, we drag & drop the newly created prefab from the *Unity Project* tab into the *Unity Inspector* tab of our snap zone.
```

Comment:
Same as above. How to add a prefab in a inspector field is probably not necessary.